rep-I0005.rst

```diff
@@ -0,0 +1,164 @@
+::
+
+   REP: I0005
+   Title: Cartesian Motion Extensions for the Simple Message Protocol
+   Authord: Fred Proctor <proctor@nist.gov> and Tre Harrison <william.harrison@nist.gov>
+   Status: Active
```

Comment:
Again, nitpicking, but a REP can only become active once it's been accepted (which requires a voting round).